src/thunderbird_accounts/mail/views.py

```diff
@@ -672,6 +681,61 @@ def remove_custom_domain(request: HttpRequest):
     return JsonResponse({'success': True})
 
 
+@login_required
+@require_http_methods(['POST'])
+def add_email_alias(request: HttpRequest):
+    """Adds an email alias"""
+    data = json.loads(request.body)
+    email_alias = data.get('email-alias')
+    domain = data.get('domain')
+
+    if not email_alias or not domain:
+        return JsonResponse({'success': False, 'error': _('Email alias and domain are required')}, status=400)
+
+    if (
+        domain not in request.user.domains.values_list('name', flat=True)
+        and domain not in settings.ALLOWED_EMAIL_DOMAINS
+    ):
+        return JsonResponse({'success': False, 'error': _('Domain not found')}, status=404)
+
+    full_email_alias = f'{email_alias}@{domain}'
+
+    try:
+        stalwart_client = MailClient()
+        stalwart_client.save_email_addresses(request.user.stalwart_primary_email, full_email_alias)
+    except Exception as e:
+        logging.error(f'Error adding email alias: {e}')
+        return JsonResponse(
+            {'success': False, 'error': 'An error occurred while adding the email alias. Please try again later.'},
+            status=500,
+        )
+
+    return JsonResponse({'success': True})
```

Comment:
I successfully see the error! Woot.

<img width="1191" height="271" alt="Image" src="https://github.com/user-attachments/assets/3d2490d8-3ddc-427b-91df-eff05d0aaad9" />

However when I went to review, I noticed we're not adding an Email entry to the User's Account object.  We have account and email objects to keep track locally, while we don't use it directly it's useful if we need to reference or lookup addresses. 

Could you do a Emails.objects.create( ..., type=alias) here? That should also trigger the next issue I'm about to mention which is unique names. Email name has a unique constraint which will trigger a user tries to add an alias that already exists as a user. 

For an error message we should just say this email address is not available. 

Thanks!